gyrinx/core/templates/core/crew/crew_form.html

```
@@ -16,6 +16,14 @@ <h1 class="h3 mb-0">{{ crew }}</h1>
             <form method="post" class="vstack gap-3">
                 {% csrf_token %}
                 {{ form.non_field_errors }}
+                {% if form.custom_count and form.saved_pick_count > form.custom_count %}
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)